### ISSUE TITLE
UTC DateTime not serialising to spec

### ISSUE THREAD

**Issue by mcshaz** [ical-org/ical.net#56]
the date

``` C#
LastModified = new CalDateTime(DateTime.SpecifyKind(new DateTime(2010, 3, 27, 13, 53, 35), DateTimeKind.Utc))
```

is serialising as:

`LAST-MODIFIED;TZID=UTC:20100327T135335Z`

the standard states:
      The "TZID" property parameter MUST NOT be applied to DATE-TIME properties whose time values are specified in UTC.

(This was found using the test in my 'experimental' branch posting a serialised calendar object to icalendar.org for validation)

**Comment by mcshaz** [comment]
fixed and tested as of commit referencing this issue in pull request #54

**Comment by rianjs** [comment]
If it's fixed, want to close this up? (Please reference the commit SHA that fixes it as your close comment.)

**Comment by mcshaz** [comment]
fixed d30880a42f43794d5274e90c9d04e918b134b238
